elasticsearch/connection/base.py

```diff
@@ -123,5 +123,3 @@ def _raise_error(self, status_code, raw_data):
             logger.warning('Undecodable raw error response from server: %s', err)
 
         raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)
-
-
```

Comment:
please configure your editor not to make changes like these. It makes the PR hard to review when they are unrelated stylistic changes in unrelated places.